esmvaltool/recipes/hydrology/recipe_globwat.yml

```diff
@@ -0,0 +1,98 @@
+# ESMValTool
+# globwat.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the GlobWat hydrological model.
+    GlobWat is a Global Hydrological Model provided by FAO
+    http://www.fao.org/nr/water/aquamaps/
+
+    By default, PET is calculated by the De Bruin method. However, users
+    can choose the Langbein method by setting the langbein_pet in the recipe
```

Comment:
Same for Langbein

Reply:
The year of publication added behind the method name.